doc/api/errors.md

```diff
@@ -2073,6 +1990,64 @@ removed: v10.0.0
 Used when an attempt is made to use a `zlib` object after it has already been
 closed.
 
+<a id="unreleased_error_codes">
```

Comment:
It seems this anchor is unneeded (as well as the `<a id="legacy-nodejs-error-codes"></a>` above). We use hardcoded anchors only for error headings in this doc.

Reply:
done. Also, any other suggestions welcome, for the last section description. The current `These errors have never been released, but had been present on master between releases.` seems to assume knowledge of the git repo on the behalf of a reader.